understanding/20/audio-description-or-media-alternative-prerecorded.html

```diff
@@ -58,23 +61,22 @@ <h2>Intent of Audio Description or Media Alternative (Prerecorded)</h2>
          
          <p>
             For 1.2.3, 1.2.5, and 1.2.7, if all of the information in the video track is already
-            provided in the audio track, no audio description is necessary. 
-            
-            
+            integrated into and provided by the main soundtrack, it is not necessary to add audio
+            description.
```

Comment:
This is a key point, but it's no good only having it in the "understanding" section. Hence my edits to the normative sections earlier in this PR, which state this explicitly.